taliforth.asm

```diff
@@ -636,15 +636,14 @@ _no_underflow:
 ; =====================================================================
 ; PRINTING ROUTINES
 
-; We distinguish two types of print calls, both of which take the string number
-; (see strings.asm) in A:
+; print_string_no_lf prints a string from string_table (see strings.asm)
+; indexed by the accumulator, with no trailing line ending.
 
-;       print_string       - with a line feed
```

Comment:
print_string was never used so I removed it, and likewise print_common is never called except from here.  Not sure if you prefer to keep those as useful routines for external use?